src/parser/mage/shared/modules/items/IreOfTheAscended.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import Statistic from 'interface/statistics/Statistic';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+
+const DAMAGE_BONUS: {[rank: number]:number } = {
+  1: .02,
+  2: .024,
+  3: .028,
+  4: .032,
+  5: .036,
+  6: .04,
+  7: .044,
+  8: .048,
+  9: .052,
+  10: .056,
+  11: .06,
+  12: .064,
+  13: .068,
+  14: .072,
+  15: .08,
+};
+
+class IreOfTheAscended extends Analyzer {
+
+  bonusDamage = 0;
+
+  constructor(props: any) {
+    super(props);
+    this.active = this.selectedCombatant.hasConduitBySpellID(SPELLS.IRE_OF_THE_ASCENDED.id);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.onDamage);
+  }
+
+  onDamage(event: DamageEvent) {
+    if (!this.selectedCombatant.hasBuff(SPELLS.RADIANT_SPARK.id)) {
+      return;
+    }
+
+    const conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.IRE_OF_THE_ASCENDED.id);
+    this.bonusDamage += calculateEffectiveDamage(event, DAMAGE_BONUS[conduitRank]);
```

Comment:
Just a comment for the future; once we can get some real testing done we should see if this applies to all damage or only to specific spells (like the class balance aura)

Reply:
I believe radiant spark counts anything the player does. But that's something we can look into